spec/commands/exec_spec.rb

```diff
@@ -851,7 +851,7 @@ def bin_path(a,b,c)
         bundle :install, :system_bundler => true, :path => "vendor/bundler"
       end
 
-      it "overrides disable_shared_gems so bundler can be found", :ruby_repo, :rubygems => ">= 2.6.2" do
```

Comment:
Is it collect? (We will support only >= RG 2.5.2)

Reply:
I tried to run this spec on rubygems 2.5.2 and it passed, so I assumed this was a typo in the filter... :man_shrugging:.

Reply:
👍  I see.